make/test.mk

```diff
@@ -120,6 +120,17 @@ ifeq ($(HOST_REPO_PATH),)
 	$(MAKE) prepare-e2e-repo E2E_REPO_PATH=$(HOST_REPO_PATH) REPO_NAME=host-operator
 endif
 
+.PHONY: get-registration-service-repo
+get-registration-service-repo:
+ifeq ($(REG_REPO_PATH),)
+	$(eval REG_REPO_PATH = /tmp/codeready-toolchain/registration-service)
+	rm -rf ${REG_REPO_PATH}
+	# clone
+	git clone https://github.com/codeready-toolchain/registration-service.git ${REG_REPO_PATH}
```

Comment:
Can we do shallow clone here? something like `git clone -–depth 1 https://github.com/codeready-toolchain/registration-service.git` to improve performance.

Reply:
Done https://github.com/codeready-toolchain/toolchain-e2e/pull/12/commits/1b50da335d5059ac5991579812ea9315b1e26beb for all clone we are making in e2e test.

Reply:
We should be careful with that. The repos we clone are used in two use cases:
- Against master (then we do not need any history)
- Against particular branch. In this case we will switch to the PR branch (which can have multiple commits) of the repo and it may be broken if we do shallow clone.

Let's not introduce shallow clones in this PR. Let's discuss/address it separately.

Reply:
@alexeykazakov good input, reverted `--depth 1` changes  https://github.com/codeready-toolchain/toolchain-e2e/pull/12/commits/927732311359236e90bce45a6eac64c851823cdd